FilmeApp/components/ExplorePage.js

```diff
@@ -46,6 +46,14 @@ export default class ExplorePage extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    BackHandler.removeEventListener('hardwareBackPress', this.handleBackPress);
+  }
+
+  handleBackPress = () => {
+    BackHandler.exitApp();
```

Comment:
this.backHandler

Reply:
this.backHandler does not have exitApp(), therefore we will need this line as it is